.asf.yaml

```diff
@@ -64,6 +64,10 @@ github:
   ghp_branch: gh-pages
   ghp_path: /
 
+  collaborators:
+    # Adding renovate-bot as a collaborator, so CI doesn't need to be manually approved
+    - renovate-bot
```

Comment:
Do we need to also add to `.github/CODEOWNERS` ?

Reply:
mean - i hope not

Reply:
Let's try 😄